pkg/pillar/zedUpload/datastore_aws.go

```diff
@@ -193,6 +195,35 @@ func (ep *AwsTransportMethod) processS3Download(req *DronaRequest) (error, int)
 	return err, csize
 }
 
+func (ep *AwsTransportMethod) processS3DownloadByChunks(req *DronaRequest) error {
+	pwd := strings.TrimSuffix(ep.apiKey, "\n")
+	prgChan := make(zedAWS.ChunkStatsChan)
```

Comment:
If prgChan means progress it is presumably misnamed since it seems like ChunkStatsChan is misnamed.